test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -94,7 +94,7 @@ var _ = SIGDescribe("Upgrade [Feature:Upgrade]", func() {
 
 			testSuite := &junit.TestSuite{Name: "Master upgrade"}
 			masterUpgradeTest := &junit.TestCase{
-				Name:      "[sig-cluster-lifecycle] master-upgrade",
+				Name:      "[sig-cloud-provider-gcp] master-upgrade",
```

Comment:
is this the right tag now? /shrug

Reply:
we didn't want `sig-cloud-provider` and gcp is a sub-project so there is no valid tag for this
https://github.com/kubernetes/kubernetes/pull/83744#discussion_r333694270

Reply:
ok, just not sure what the pattern is, I'll drop a /approve and let someone else bikeshed ;-)

Reply:
these are mostly for the visibility of the release team and triage.